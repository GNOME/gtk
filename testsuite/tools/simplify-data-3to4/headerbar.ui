<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <!-- interface-requires gtk+ 3.0 -->
  <object class="GtkWindow" id="window1">
    <child type="titlebar">
      <object class="GtkHeaderBar" id="headerbar1">
        <property name="visible">True</property>
        <property name="show-close-button">True</property>
        <property name="custom-title">box1</property>
      </object>
    </child>
  </object>
  <object class="GtkBox" id="box1">
    <property name="visible">True</property>
  </object>
</interface>
