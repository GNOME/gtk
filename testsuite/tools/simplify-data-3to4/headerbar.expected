<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <object class="GtkWindow" id="window1">
    <child type="titlebar">
      <object class="GtkHeaderBar" id="headerbar1">
        <property name="title-widget">box1</property>
      </object>
    </child>
  </object>
  <object class="GtkBox" id="box1"/>
</interface>
